### PR TITLE
Fn products python

### PR DESCRIPTION
**Pull request by Codeyenum**
Completed the products(python) page

### CODE REVIEW

**Review by Lharbeeb — APPROVED** [review]

File: frontend/css/products-python.css [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+    font-size: 1.5rem;
+}
+
+h2 {
+    font-size: 1.2rem;
+}
+
+h3 {
+    font-size: 1.1rem;
+}
```
Comment (spykelionel) [inline frontend/css/products-python.css:176]: Must not style global elements. Give them appropriate class names.

**Review by spykelionel — CHANGES_REQUESTED** [review]
Global styles used in css. Give appropriate class names

File: frontend/css/products-python.css (hunk 1 above)
Comment (Codeyenum) [reply frontend/css/products-python.css]: Done

File: frontend/css/products-python.css [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+body {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    font-family: 'Open Sans', sans-serif;
+    min-height: 100vh;
+    padding: 0 1.25rem;
+    text-align: center;
+}
```
Comment (spykelionel) [inline frontend/css/products-python.css:20]: These styles are not specific to your page. use class names here.
the font-size: 18px in the html can be moved to body.

**Review by spykelionel — CHANGES_REQUESTED** [review]
Check comments in css file

File: frontend/css/products-python.css (hunk 2 above)
Comment (spykelionel) [reply frontend/css/products-python.css]: Remove the global * in your css

**Pull request closed without merging**